buildlib/azure-pipelines-pr.yml

```diff
@@ -47,6 +47,32 @@ stages:
                  echo "##vso[task.complete result=Failed;]"
               fi
             condition: eq(variables['Build.Reason'], 'PullRequest')
+  - stage: Format
+    jobs:
+      # Check that the code is formatted according to the code style guidelines
+      - job: format
```

Comment:
can we make "clang-format" job pat of "stage: Codestyle" ?

Reply:
Done